src/Optimization/hiopAlgPrimalDecomp.cpp

```diff
@@ -158,9 +174,9 @@ HessianApprox(hiopInterfacePriDecProblem* priDecProb,
               hiopOptions* options_pridec,
               MPI_Comm comm_world)
   : HessianApprox(-1, 
-		  priDecProb, 
-		  options_pridec, 
-		  comm_world)
+                  priDecProb, 
```

Comment:
can you do a 1-liner for `HessianApprox` ?

Reply:
Yes.